source/Calamari/Kubernetes/Commands/Executors/BaseKubernetesApplyExecutor.cs

```diff
@@ -135,12 +135,15 @@ void WriteResourcesToOutputVariables(IEnumerable<ResourceIdentifier> resources)
             }
         }
         
-        void LogParsingError(string outputJson)
+        void LogParsingError(string outputJson, bool logKubectlOutputOnError)
         {
-            log.Error($"\"kubectl apply -o json\" returned invalid JSON:");
-            log.Error("---------------------------");
-            log.Error(outputJson);
-            log.Error("---------------------------");
+            log.Error($"\"kubectl apply -o json\" returned invalid JSON");
+            if (logKubectlOutputOnError)
```

Comment:
I've added this in as it feels like this could have similar security concerns to the other one. Do we want to put in the (visible by default) error message that they can switch this flag on to log the raw kubectl output?